.eslintrc.js

```diff
@@ -0,0 +1,330 @@
+        "sort-vars": "error",                        // requires all variable declaration blocks and verifies that all variables are sorted alphabetically
+        "space-before-blocks": "off",                // enforce consistency of spacing before blocks. It is only applied on blocks that don’t begin on a new line
+        "space-before-function-paren": "off",        // require or disallow a space before function parenthesis
+        "space-in-parens": [                         // enforce consistent spacing directly inside of parentheses
+            "error",
+            "never"                                  // never: enforces zero spaces inside of parentheses
+        ],
+        "space-infix-ops": "off",                    // require spacing around infix operators
+        "space-unary-ops": [                         // enforces consistency regarding the spaces after words unary operators
+            "error",
+            { "words": true }
+        ],
+        "spaced-comment": [                          // enforce consistency of spacing after the start of a comment // or /*
+            "error",
+            "always"
+        ],
+        "strict": [                                  // requires or disallows strict mode directives
+            "error",
+            "never"                                  // never: disallows strict mode directives
+        ],
+        "switch-colon-spacing": "error",             // controls spacing around colons of case and default clauses in switch statements. Default: { "after": true, "before": false}
+        "symbol-description": "error",               // requires a description when creating symbols
+        "template-curly-spacing": [                  // aims to maintain consistency around the spacing inside of template literals
+            "error",
+            "never"                                  // never: disallows spaces inside of the curly brace pair
+        ],
+        "template-tag-spacing": "error",             // aims to maintain consistency around the spacing between template tag functions and their template literals. Default: never
+        "unicode-bom": [                             // require or disallow the Unicode Byte Order Mark
+            "error",
+            "never"                                  // never: files must not begin with the Unicode BOM
+        ],
+        "vars-on-top": "error",                      // require variable declarations to be at the top of their scope
+        "wrap-iife": "error",                        // requires all immediately-invoked function expressions to be wrapped in parentheses
+        "wrap-regex": "error",                       // requires regex expressions to be wrapped using parentheses to make it more readable
+        "yield-star-spacing": "error",               // enforces spacing around the * in yield* expressions. Default: {"before": false, "after": true}
```

Comment:
nit: I think this could be clearer
```suggestion
        "yield-star-spacing": "error",               // enforces spacing around the '*' in 'yield*' expressions. Default: {"before": false, "after": true}
```